### PR TITLE
Closes #172 Updates to align with updates to tidyselect and dplyr functionality

### LINKED ISSUE #172: Bug: Warning message "Using `across()` without supplying `.cols` was deprecated in dplyr 1.1.0."
### What happened?

When I run axecute(), I get a warning message as shown below:

logrx::axecute("xxxxxxx.R", log_name = "xxxxxxx.log", log_path = "xxxxxxx")
── Attaching core tidyverse packages ──────────────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.0     ✔ readr     2.1.4
✔ forcats   1.0.0     ✔ stringr   1.5.0
✔ ggplot2   3.4.1     ✔ tibble    3.2.0
✔ lubridate 1.9.2     ✔ tidyr     1.3.0
✔ purrr     1.0.1     
── Conflicts ────────────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package to force all conflicts to become errors
2023-04-27 14:29:04 EDT: Reading xxxxxxx.sas7bdat (Last modified 2023-04-21 14:42:58 EDT)
2023-04-27 14:29:04 EDT: Writing xxxxxxx.xpt (Version 5)
Warning: There was 1 warning in `distinct()`.
ℹ In argument: `across()`.
Caused by warning:
! Using `across()` without supplying `.cols` was deprecated in dplyr 1.1.0.
ℹ Please supply `.cols` instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated.

### Session Information

> sessionInfo()
R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Red Hat Enterprise Linux

Matrix products: default
BLAS:   /usr/lib64/libblas.so.3.4.2
LAPACK: /usr/lib64/liblapack.so.3.4.2

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       
 [4] LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
[10] LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] grid      stats     graphics  grDevices datasets  utils     methods   base     

other attached packages:
 [1] gtable_0.3.1    haven_2.5.2     lubridate_1.9.2 forcats_1.0.0   stringr_1.5.0  
 [6] dplyr_1.1.0     purrr_1.0.1     readr_2.1.4     tidyr_1.3.0     tibble_3.2.0   
[11] ggplot2_3.4.1   tidyverse_2.0.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.10       waiter_0.2.5      ps_1.7.2          rprojroot_2.0.3  
 [5] digest_0.6.31     logrx_0.2.1       utf8_1.2.3        mime_0.12        
 [9] R6_2.5.1          pillar_1.8.1      rlang_1.1.0       lazyeval_0.2.2   
[13] rstudioapi_0.14   miniUI_0.1.1.1    callr_3.7.3       desc_1.4.2       
[17] labeling_0.4.2    munsell_0.5.0     slushy_0.2.6      shiny_1.7.4      
[21] compiler_4.2.1    httpuv_1.6.9      lintr_3.0.2       pkgconfig_2.0.3  
[25] tcltk_4.2.1       htmltools_0.5.4   tidyselect_1.2.0  fansi_1.0.4      
[29] crayon_1.5.2      tzdb_0.3.0        withr_2.5.0       later_1.3.0      
[33] jsonlite_1.8.4    xtable_1.8-4      lifecycle_1.0.3   magrittr_2.0.3   
[37] scales_1.2.1      cli_3.6.0         stringi_1.7.12    farver_2.1.1     
[41] renv_0.16.0       promises_1.2.0.1  remotes_2.4.2     rex_1.2.1        
[45] xml2_1.3.3        ellipsis_0.3.2    generics_0.1.3    vctrs_0.5.2      
[49] cyclocomp_1.1.0   tools_4.2.1       glue_1.6.2        hms_1.1.2        
[53] processx_3.8.0    fastmap_1.1.1     timechange_0.2.0  colorspace_2.1-0 
[57] sessioninfo_1.2.2

### Reproducible Example

_No response_

### PR DESCRIPTION
**Pull request by kodesiba**
# Release Description
Hotfix release

## Milestone
Milestone: N/A
Closes #172 

# Release Checklist

- [x] Version number has been updated
- [x] Description file updated with New Developers (if applicable)
- [x] NEWS.md has been updated
- [x] README.md has been updated (if applicable)
- [x] Vignettes have been updated (if applicable)
- [x] Ensure all unit tests are passing
- [x] GitHub actions on this PR are all passing
- [x] Run `spelling::spell_check_package()` and address any issues
- [x] Draft GitHub release created using automatic template and updated with additional details. Remember to click "release" after PR is merged.

### CODE REVIEW

**Comment by parmsam-pfizer** [comment]
Do we need `distinct(across())` or `distinct(across(.cols = everything(), fns = NULL))`? It looks like `distinct()` would suffice since it gets distinctness using all variables in the dataframe. Alternatively, if we need to include the tidyselect part then this works: `distinct(across(.cols = everything()))`
https://github.com/pharmaverse/logrx/pull/174/files#diff-be75f6398b6213cfe50e1318d8488ba76de0c0cefddbaae032844bad84fe8492R200-R201

**Comment by kodesiba** [comment]
> 

@nicholas-masel  any thoughts here?  Not fully sure how everything is working in this function and I think you were the one who wrote this part.

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# logrx 0.2.2
+
+ - hotfix for update to `across()` and `.data$var` updates in source packages (#172)
```
Comment (bms63) [inline NEWS.md:3]: mention requirement for dplyr

**Review by nicholas-masel — CHANGES_REQUESTED** [review]
@parmsam-pfizer is correct.  This can just use distinct() without across().

**Review by bms63 — APPROVED** [review]

**Pull request merged**